src/Model/GoMod.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace AutoRest.Go.Model
+{
+    public class GoMod
+    {
+        public GoMod(string goMod)
+        {
+            Module = goMod;
+        }
+
+        public string Module { get; }
```

Comment:
I guess it doesnt really matter for now, but should this be readonly also ?

Reply:
I guess it doesnt really matter for now, but should this be readonly also ?

Reply:
It is read-only by virtue of no setter, or am I misunderstanding?

Reply:
yea its fine, the github extension published the comment although I thought it didnt, even twice  :/